dbms/src/TiDB/Schema/TiDBSchemaSyncer.cpp

```diff
@@ -84,10 +84,17 @@ bool TiDBSchemaSyncer<mock_getter, mock_mapper>::syncSchemasByGetter(Context & c
     }
     else
     {
+#if SERVERLESS_PROXY == 0
         if (version <= cur_version)
         {
             return false;
         }
+#else
+        if (version == cur_version)
```

Comment:
If comes from https://github.com/tidbcloud/tiflash-cse/pull/70/files. Do you remember why this change happened?

Reply:
In TiFlash master, tidb execute `flashback cluster` won't overwrite the schema_version.
But in cse branch, they may restore the cluster by the data from S3. And this will lead to schema version rollbacked to a smaller version.